<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KBEditor</class>
 <widget class="QDialog" name="KBEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KB Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2" colspan="2">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="minimumSize">
      <size>
       <width>398</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>398</width>
       <height>28</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:600; color:#ff0000;&quot;&gt;RED&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt;&quot;&gt; means that samples exist in the main KB!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="9">
    <widget class="QLineEdit" name="Le_KBName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="Btn_LoadKB">
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Load KB</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="Btn_SaveSample">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Save sample</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="9">
    <widget class="QPlainTextEdit" name="pTE_EditSample">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="9">
    <widget class="QListWidget" name="Lw_KB">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView::item:selected {
     color: rgb(0, 0, 0);
     border: 1px solid black;
	 border-radius: 0px;
     background-color: rgb(0, 220, 0); 
     text-align: bottom;
    }

QListView::item {
   //  border: 1px solid black;
	// border-radius: 0px;
   //  background-color: rgb(220, 220, 220);
    // text-align: center;
    }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="7" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="Btn_SendSamples">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>166</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>166</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Send sample to main KB</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="Btn_Header">
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Header</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="Btn_DeleteSample">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>108</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>108</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Delete selected</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="Btn_AddSample">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Add sample</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="Btn_CreateKB">
     <property name="text">
      <string>Create KB</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="9">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="Btn_LoadAllFromMainKB">
     <property name="minimumSize">
      <size>
       <width>142</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>142</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Load all from main KB</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KBEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KBEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
